sport/app/football/controllers/context.scala

```diff
@@ -53,7 +53,7 @@ trait CompetitionListFilters {
 trait CompetitionTableFilters {
   def competitionsService: CompetitionsService
   def filters: Map[String, Seq[CompetitionFilter]] =
-    competitionsService.competitionsWithTodaysMatchesAndFutureFixtures.competitions
```

Comment:
This check was removing the Scottish Championship. The filters here were used for the dropdown menu but not for the tables on the page. This change moves this filter to being closer to what is done to choose the displayed tables